django/db/backends/sqlite3/base.py

```diff
@@ -27,13 +30,30 @@
 from .schema import DatabaseSchemaEditor                    # isort:skip
 
 
+if settings.DEBUG:
+    sqlite3.enable_callback_tracebacks(True)
+else:
+    sqlite3.enable_callback_tracebacks(False)
+
+
 def decoder(conv_func):
     """
     Convert bytestrings from Python's sqlite3 interface to a regular string.
     """
     return lambda s: conv_func(s.decode())
 
 
+def none_guard(func):
```

Comment:
Wondering if `null_guard` is a better name as this is emulating a SQL function?

Reply:
None is better since arg values that  are checked is in python.

Reply:
Fair enough.